.github/workflows/create_github_release.yml

```diff
@@ -32,15 +32,13 @@ jobs:
     steps:
       - uses: actions/checkout@v4
 
-      # Create the Release TAG and notes.
-      - name: Create GitHub Release
-        uses: actions/create-release@v1
+      # Create a Github release with release notes.
+      - name: Create Github release
+        uses: ncipollo/release-action@v1
         env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
I checked and it worked without needing to pass the token, probably they both have the same value in our context. But I agree that it would be good to pass the token. Will add it.